[2025-08-26 15:36:02,255] [INFO] | Scheduler initialized — runs every Friday at 06:00 AM.
[2025-08-26 15:36:02,255] [INFO] | Immediate startup run triggered...
[2025-08-26 15:36:02,255] [INFO] | Pipeline started.
[2025-08-26 15:36:02,255] [INFO] | Launching crawler — collecting fresh documents...
[2025-08-26 15:40:14,701] [WARNING] | Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'ConnectionResetError(54, 'Connection reset by peer')': /session/52c446688f01f7f5345d3fb98b618c1c
[2025-08-26 15:40:14,702] [WARNING] | Retrying (Retry(total=1, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x10d431bd0>: Failed to establish a new connection: [Errno 61] Connection refused')': /session/52c446688f01f7f5345d3fb98b618c1c
[2025-08-26 15:40:14,712] [WARNING] | Retrying (Retry(total=0, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x10d431e50>: Failed to establish a new connection: [Errno 61] Connection refused')': /session/52c446688f01f7f5345d3fb98b618c1c
